juju/model.py

```diff
@@ -176,6 +216,9 @@ class ModelBackend:
     def _run(self, *args):
         return json.loads(run(args + ('--format=json',), stdout=PIPE, check=True).stdout.decode('utf8'))
```

Comment:
Not about this PR, but this line would benefit from being broken down. There's way too much happening there, and little reason for it to be packed in that single line.